src/components/Forms/Login/LoginForm.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { render, RenderResult } from "@testing-library/react";
+import React from "react";
+
+import { ILoginForm } from "../../../models";
+import LoginForm from "./LoginForm";
+
+const intialFormValue: ILoginForm = {
+	email: "",
+	password: "sadas",
+};
+const onSubmitMock = jest.fn();
+const clearError = jest.fn();
+const validate = jest.fn();
+
+let loginForm: RenderResult<typeof import("/mnt/c/Users/Shadab/Repositories/fundimpact-frontend/node_modules/@types/testing-library__dom/queries")>;
```

Comment:
You have provided a local path here. It would result in error in other machine

Reply:
Done